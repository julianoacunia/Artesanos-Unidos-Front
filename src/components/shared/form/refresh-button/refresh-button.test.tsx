import * as React from 'react';
import renderer from 'react-test-renderer';
import RefreshButton from './refresh-button';

describe('Submit button', () => {
  test('should match snapshot and styles', () => {
    const tree = renderer
    .create(<RefreshButton
      testId={'test'}
      text={'test'}
      onClick={() => 'test'}
    />)
    .toJSON();
      expect(tree).toMatchSnapshot();
  });
});