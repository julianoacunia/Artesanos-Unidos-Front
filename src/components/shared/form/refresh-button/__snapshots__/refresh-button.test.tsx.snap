// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Submit button should match snapshot and styles 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-contained button submitButton"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  test-id="test"
  type="button"
>
  <span
    className="MuiButton-label"
  >
    <span
      className="MuiButton-startIcon MuiButton-iconSizeMedium"
    >
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z"
        />
      </svg>
    </span>
    test
  </span>
</button>
`;
